lib/rest.js

```diff
@@ -90,7 +88,7 @@ module.exports = class Rest {
                 page++;
                 shouldPaginate = true;
                 if (this.options?.eventHandlers?.onLoop) {
-                    this.options.eventHandlers.onLoop(null, collector?.[iteratorField]);
+                    this.options.eventHandlers.onLoop(undefined, collector?.[iteratorField]);
```

Comment:
why? that is a potentially breaking change for users of the SDK

Reply:
good point, put a todo to replace when moving to v1 ( potential breaking change)